staging/src/k8s.io/kubectl/docs/book/pages/app_composition_and_deployment/diffing_local_and_remote_resources.md

```diff
@@ -33,6 +33,7 @@ The `KUBECTL_EXTERNAL_DIFF` environment variable can be used to select your own
 By default, the "diff" command available in your path will be run with "-u" (unified) and "-N"
 (treat new files as empty) options.
 
+If the exit code after diff command is 0, means there is no change detected and the exit code 1 means there is at least one change which indicates `KUBECTL_EXTERNAL_DIFF` programs must also follow the convention of using 1 as their exit code for changes detected. If the exit code is more than 1, means error occurs in this command.
```

Comment:
Some of diff man-pages are presented like this:
```
EXIT STATUS
The following exit values shall be returned:

 0
No differences were found.
 1
Differences were found.
>1
An error occurred.
```

which seems much simpler.

Here's my suggestion:
```
EXIT STATUS
The following exit values shall be returned:

 0
No differences were found.
 1
Differences were found.
>1
Kubectl or diff failed with an error.

Note: `KUBECTL_EXTERNAL_DIFF`, if used, is expected to follow that convention.
```

What do you think?

Reply:
We might want to document this in the kubectl command directly too, thanks!

Reply:
Good, thanks! I made changes.